draft-ietf-cose-bls-key-representations.md

```diff
@@ -462,24 +422,21 @@ Below is the above CBOR rendered in diagnostic view.
 {
   1: 1,
   -1: 14,
-  -2: h'03B81526B001B41701632609EE1285CF09549E2228CAA0775A052AE6F1AE5737
+  -2: h'TBD THIS IS NOW WRONG. REPLACE IT. 49E2228CAA0775A052AE6F1AE5737
 DCC66A8C0F0B65FAC070D85B0217CC6C0CE5DB1C27767EF73F2E8355B5F02DABA2B81054
 8FC8F0BE26C329375B95B6B6D844CE981CED5934709E57ACA986F927',
-  -3: h'1097125DA5EBD6B31D875A3884C2197B1AFCDECBBBBC26871E38880FE80EAC09
-C6E754E865A9F01B3ED511D86673A6D8109DC94D0840FF539FA2D175B2DEA6BD3484D91A
-73424AD4C4ACC9CC748FB5280477A5C187DC6D749D0DE00BEEE1A206',
-  -4: h'2A9CFF87262CE1B093032446AC6911C4B947305DEEA7631E23D1A27C46BD9F62'
+  -4: h'TBD THIS IS NOW WRONG. REPLACE IT. 7305DEEA7631E23D1A27C46BD9F62'
```

Comment:
```suggestion
  -2: h'b03ea8b181c4a4d3c5927a9ae3ad98587566b0dba9008240f5d54db7b19ef15c09
2e8baf88394a3fb52f063a6f56488a18e58f81d5ee1bf6c5146adcb1f43d746791322ee275
b022de06ed8b3f8ab56e53a1f6943662b3df528a5640a03eec6a',
  -4: h'3fe7a731678b27201792c20fb11fd91130edc405b8647cfd1ce4598e7186680d'
```

Reply:
`x` and `d` are consistent in my testing. Would it make sense for this to be the same key as the corresponding JWK example, though?